sdk/data/aztables/testdata/perf/insert_entity.go

```diff
@@ -164,8 +164,8 @@ func (g *insertEntityTestGlobal) NewPerfTest(ctx context.Context, options *perf.
 }
 
 func (d *insertEntityPerfTest) Run(ctx context.Context) error {
-	_, err := d.tableClient.InsertEntity(ctx, d.entity, &aztables.InsertEntityOptions{
-		UpdateMode: aztables.EntityUpdateModeMerge,
+	_, err := d.tableClient.UpsertEntity(ctx, d.entity, &aztables.UpsertEntityOptions{
```

Comment:
UpsertEntity is the preferred name in all languages?

Reply:
I don't know if it's the preferred name, but it's what we GA'ed with so we can't change it now.